app/views/general_shopping_lists/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= stylesheet_link_tag 'foods', media: 'all', 'data-turbolinks-track': 'reload' %>
+<%= stylesheet_link_tag 'recipe_details', media: 'all', 'data-turbolinks-track': 'reload' %>
+<h1>Shopping List</h1>
+
+<div class= 'amount-price'>
+  <p>Amount of food items to buy : <%= @missing_foods.count %> <p>
+  <p>total value of food needed: <%= @sum %></p>
+</div>
+
+<section class='shopping-table'>
+<table class='table food-table shopping-t'>
+    <tr>
+      <th class='table-data'>Food</th>
+      <th class='table-data'>Quantity</th>
+      <th class='table-data'>Price</th>
+    </tr>
+    <% @missing_food_records.each do |food| %>
+      <tr>
+        <td class='table-data'><%= food.name %></td>
+        <td class='table-data'><%= "#{food.quantity} #{food.measurement_unit}" %></td>
+        <td class='table-data'><%= "$#{@food_prices[food.id]}" %></td>
+      </tr>
+    <% end %>
+  </table>
+</section>
```

Comment:
- [ ] It will be great to ensure that the general shopping list is working as expected. When I click on generate a shopping list, the page does not show the values although the structure is there. Kindly ensure that the general shopping list view achieves the following ⬇️ 

- Should show the list of food that is missing for all recipes of the logged-in user.
- Should count the total food items and total price of the missing food.

Reply:
![image](https://github.com/yashodhicy/RecipeApp/assets/116276432/3c3685d1-c3db-49a4-beed-a8a2d180150f)
It is showing from my side

Reply:
Sorry about this. This has been implemented well after a second look.

Reply:
Thank you for your kindness and sincere look on this. You are a good listner.